routes/per_capita_consumption.py

```diff
@@ -2,15 +2,15 @@
 from flask import Blueprint, jsonify
 from flask_api import status
 from flask_jwt_extended import get_jwt_identity
-from models.pkv import PKV
+from models.pcc import PerCapitaConsumption
```

Comment:
See above, please consider using a more telling name for the module instead of another abbreviation.